tests/unit/aiplatform/test_training_jobs.py

```diff
@@ -1070,6 +1071,611 @@ def test_run_call_pipeline_service_create_distributed_training(
         assert job.state == gca_pipeline_state.PipelineState.PIPELINE_STATE_SUCCEEDED
 
 
+class TestCustomContainerTrainingJob:
+    def setup_method(self):
+        importlib.reload(initializer)
+        importlib.reload(aiplatform)
+
+    def teardown_method(self):
+        initializer.global_pool.shutdown(wait=True)
```

Comment:
Should we include this teardown in future tests?

Reply:
Yes it should be included if the tests include `sync=False` and used the global_pool.